kivymd/uix/backdrop.py

```diff
@@ -328,7 +328,15 @@ def open(self, open_up_to=0):
             self.close()
             return
         if open_up_to:
-            y = open_up_to
+            # added open_up_to check
+            if open_up_to < (
+                self.ids.header_button.height - self.ids._front_layer.height
+            ):
+                y = self.ids.header_button.height - self.ids._front_layer.height
```

Comment:
This check will always be false.

Reply:
No it will not. If you do .open(-1000) or some large value that is greater then the size of the _front_layer it goes in there and limits bad animations. In this instance it is the front_layer completely falling off the screen.